src/modules/users/users.service.ts

```diff
@@ -762,10 +760,8 @@ export class UserService {
         data: { applications: userApplication, total },
       });
     } catch (error) {
-      return new ErrorResponse({
-        statusCode: HttpStatus.INTERNAL_SERVER_ERROR,
-        errorMessage: 'Failed to fetch user applications',
-      });
+      console.error('Error while fetching user applications:', error);
```

Comment:
@Xitija Do you know if removing `statusCode` from api response is breaking UI anywhere?